packages/backend/index.js

```diff
@@ -1,5 +1,7 @@
+//Do you need an app.js AND index.js? I would merge them into a single module.js
```

Comment:
Good suggestion, but I think this was done for him that is a good suggestion, but the deployments live off this file, so I am keeping it.